### PR TITLE
Add documentation for preview datasets in Kedro-viz 

### PR DESCRIPTION
**Pull request by rashidakanchwala**
> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to a separate `kedro-datasets` package in
> [`kedro-plugins` repository](https://github.com/kedro-org/kedro-plugins). Any changes to the dataset implementations
> should be done by opening a pull request in that repository.
## Description
Add documentation for preview datasets in Kedro-viz 

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: docs/source/visualisation/preview_datasets.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+```bash
+kedro new --starter=spaceflights
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+
+When your project is ready, navigate to the root directory of the project.
+
+### Configure the Data Catalog
```
Comment (astrojuanlu) [inline docs/source/visualisation/preview_datasets.md:22]: Docs build failing with

```
sphinx.errors.SphinxWarning: /home/docs/checkouts/readthedocs.org/user_builds/kedro/checkouts/2773/docs/source/visualisation/preview_datasets.md:22:Non-consecutive header level increase; H1 to H3 [myst.header]
```

So:

```suggestion
## Configure the Data Catalog
```

File: docs/source/visualisation/preview_datasets.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  metadata:
+    kedro-viz:
+      layer: raw
+      preview_args:
+          nrows: 15
+```
+
+
+
+### Previewing Data on Kedro-viz
```
Comment (astrojuanlu) [inline docs/source/visualisation/preview_datasets.md:63]:
```suggestion
## Previewing Data on Kedro-viz 
```

(See above)

**Review by astrojuanlu — COMMENTED** [review]
I didn't look at this in detail yet, commented on some whitespace and formatting issues

File: docs/source/visualisation/preview_datasets.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Preview data in Kedro-Viz
+
+This page describes how to preview data from different datasets in a Kedro project with Kedro-Viz. Kedro-viz currently supports preview of `CSVDatasets` and `ExcelDatasets`
```
Comment (stichbury) [inline docs/source/visualisation/preview_datasets.md:3]: > This page describes how to preview data from different datasets in a Kedro project with Kedro-Viz. Kedro-viz currently supports preview of `CSVDatasets` and `ExcelDatasets`

Please can you say what "current" is by saying something like: "Dataset preview was first introduced in Kedro-Viz version <whatever>, which offers preview for `CSVDatasets` and `ExcelDatasets`."

File: docs/source/visualisation/preview_datasets.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+
+We use the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) and demonstrate how to add data preview for the `customer`, `shuttle` and `reviews` datasets. Even if you have not yet worked through the tutorial, you can still follow this example; you'll need to use the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) to generate a copy of the project with working code in place.
```
Comment (stichbury) [inline docs/source/visualisation/preview_datasets.md:6]:
```suggestion
We use the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) to demonstrate how to add data preview for the `customer`, `shuttle` and `reviews` datasets. Even if you have not yet worked through the tutorial, you can still follow this example; you'll need to use the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) to generate a copy of the project with working code in place.
```

File: docs/source/visualisation/preview_datasets.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+kedro new --starter=spaceflights
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+
+When your project is ready, navigate to the root directory of the project.
+
+## Configure the Data Catalog
+
+Kedro-viz currently supports preview of two types of datasets:
```
Comment (stichbury) [inline docs/source/visualisation/preview_datasets.md:24]:
```suggestion
Kedro-Viz version whatever_it_is currently supports preview of two types of datasets:
```

File: docs/source/visualisation/preview_datasets.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  metadata:
+    kedro-viz:
+      layer: raw
+      preview_args:
+          nrows: 15
+```
+
+
+
+## Previewing Data on Kedro-viz
```
Comment (stichbury) [inline docs/source/visualisation/preview_datasets.md:63]:
```suggestion
## Previewing Data on Kedro-viz 
```

File: docs/source/visualisation/preview_datasets.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+After you've configured the Data Catalog, you can preview the datasets on Kedro-Viz. Start Kedro-Viz by running the following command in your terminal:
+
+```bash
+kedro viz
+```
+
+The previews are shown as follows:
+
+Click on each dataset node see a small preview in the metadata panel:
```
Comment (stichbury) [inline docs/source/visualisation/preview_datasets.md:73]:
```suggestion
Click on each dataset node to see a small preview in the metadata panel:
```

**Review by stichbury — CHANGES_REQUESTED** [review]
LGTM. Just a few changes requested, mostly just small fixes to capitalisation but also please can you specify the version from which this is supported? I've made some suggestions for how to do this but you'll need to drop in the version to replace the two places I used `whatever_it_is` to represent the first version supporting the functionality.

**Review by tynandebold — APPROVED** [review]
Looking good!

File: docs/source/visualisation/preview_datasets.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+## Configure the Data Catalog
+
+Kedro-Viz version 6.3.0 currently supports preview of two types of datasets:
+
+* `pandas.CSVDataset`
+* `pandas.ExcelDataset`
+
+
+To enable the preview of these datasets, you need to add the `preview_args` attribute to the `kedro-viz` configuration under the `metadata` section in the Data Catalog:
```
Comment (NeroOkwa) [inline docs/source/visualisation/preview_datasets.md:29]: Can we also state that nrow is number of rows to be previewed?

**Review by NeroOkwa — APPROVED** [review]

**Review by stichbury — APPROVED** [review]
LGTM 🏆  Thank you!

**Review by NeroOkwa — APPROVED** [review]

**Pull request merged**